OpenRA.Mods.Common/Scripting/Global/MediaGlobal.cs

```diff
@@ -27,10 +26,12 @@ namespace OpenRA.Mods.Common.Scripting
 	public class MediaGlobal : ScriptGlobal
 	{
 		World world;
+		MusicPlaylist playlist;
```

Comment:
This and the world above should be readonly, and then a newline before the constructor please.